README.md

```diff
@@ -47,9 +47,9 @@ Please build from source for other configurations (e.g., your CPU is ARM, or you
  - On Windows, please install [Microsoft Visual C++ Redistributable](https://aka.ms/vs/16/release/vc_redist.x64.exe) if you haven't.
  - [[All releases]](https://github.com/taichi-dev/taichi/releases)
 
-## Developer Installation
+## Contributing
 
-Please follow [this doc](https://docs.taichi.graphics/lang/articles/contribution/dev_install) to learn how to build Taichi from source. Note that Taichi requires LLVM-10.0.0, and it is recommneded to use [our prebuilt LLVM libraries](https://docs.taichi.graphics/lang/articles/contribution/dev_install#installing-dependencies) for each platform.
+We welcome feedback and comments! If you would like to contribute to Taichi, please check out [CONTRIBUTING.md](CONTRIBUTING.md).
```

Comment:
We'd love to hear your comments or any of your feedback! If you would like to contribute to Taichi, please check out the [Contribution Guidelines](CONTRIBUTING.md) first.

Reply:
Off-topic: Just so you know, contributing.md only holds a link to another contribution guideline on the doc site.

Reply:
```suggestion
We'd love to hear your comments or any of your feedback! If you would like to contribute to Taichi, please check out the [Contribution Guidelines](CONTRIBUTING.md) first. 
```